docs/examples/multiple_stands.md

```diff
@@ -19,13 +27,15 @@ import pytest
 def test_one():
     assert True
 ```
+* Create another project for the CouchDB instance to store the report history.
 
 ### databases
 
-Create a separate database for each project, and use a different port for each.
-To create a database use [CouchDH instance](../documentation/database.md#couchdb-instance) instruction.
+Create a separate CouchDB Instance for each project, and use a different port for each.
 
-* Create a `couchdb.ini` file:
+To create a CouchDB Instance use [CouchDH instance](../documentation/database.md#couchdb-instance) instruction.
+
+* Create a `couchdb.ini` file in each project.
```

Comment:
Where is the bash command for the third database on port 5986?

Reply:
Added.